config/database.local.yml

```diff
@@ -1,12 +0,0 @@
-default: &default
```

Comment:
This cannot be removed, it causes Travis CI to break and needs to be here for new contributors to get setup (see the ReadMe)